<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intrepid</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="backgroundBrush">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionRead_Input_File"/>
   </widget>
   <widget class="QMenu" name="menuDesign">
    <property name="title">
     <string>Design</string>
    </property>
    <addaction name="actionMentor"/>
    <addaction name="actionMentMUX"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
    <widget class="QMenu" name="menuLabels">
     <property name="title">
      <string>Labels</string>
     </property>
     <addaction name="actionAll_2"/>
     <addaction name="actionBackbone_2"/>
     <addaction name="actionNone"/>
    </widget>
    <addaction name="actionAll"/>
    <addaction name="actionBackbone"/>
    <addaction name="actionCluster"/>
    <addaction name="separator"/>
    <addaction name="actionTree"/>
    <addaction name="actionDirect"/>
    <addaction name="separator"/>
    <addaction name="menuLabels"/>
    <addaction name="separator"/>
    <addaction name="actionNormal"/>
    <addaction name="actionUtilisation"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionStandard"/>
    <addaction name="actionCSV"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDesign"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuReport"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionRead_Input_File">
   <property name="text">
    <string>Read Input File</string>
   </property>
  </action>
  <action name="actionMentor">
   <property name="text">
    <string>Mentor</string>
   </property>
  </action>
  <action name="actionMentMUX">
   <property name="text">
    <string>MentMUX</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionBackbone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backbone</string>
   </property>
  </action>
  <action name="actionCluster">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cluster</string>
   </property>
  </action>
  <action name="actionTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tree</string>
   </property>
  </action>
  <action name="actionDirect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Direct</string>
   </property>
  </action>
  <action name="actionAll_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </action>
  <action name="actionBackbone_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Backbone</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionUtilisation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Utilisation</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
